apps/mainsite/utils.py

```diff
@@ -485,3 +485,7 @@ def convert_svg_to_png(svg_string, height, width):
     except ValueError:
         # Issuing decoding response JSON
         return False
+
+def verifyIssuerAutomatically(url, email):
+    emailDomain = email.split('@')[1]
+    return emailDomain in url
```

Comment:
This might cause wrong results if the email domain is merely a substring of the issuer domain

Reply:
Can you provide an example where it might cause a wrong result?

Reply:
I updated the implementation of extracting URL domain and matching domains.